random_user_agent/user_agent.py

```diff
@@ -14,9 +14,11 @@ class UserAgent:
         'popularity': [],
     }
 
-    def __init__(self, limit=None, *args, **kwargs):
+    def __init__(self, all =False, limit=None, *args, **kwargs):
```

Comment:
Loading all the user agents can be achieved as follows:

```
user_agent_rotator = UserAgent()
```

There is no need for the `all` parameter. We can revert these code changes.

It's better to add an update to the `README.md` mentioning how can we load all the user agents.